packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -801,7 +801,7 @@ export default class TabAdvanced extends Component {
                 <tr>
                   <th>
                     {_('autoPowerOn')}
-                    {vm.auto_poweron && !vmPool.auto_poweron && (
+                    {vm.auto_poweron && !vmPool?.auto_poweron && (
                       <Tooltip content={_('poolAutoPoweronDisabled')}>
                         <a className='btn btn-link btn-sm' onClick={() => editVm(vm, { auto_poweron: true })}>
                           {' '}
```

Comment:
Not related to this PR, but i don't understand this code.
Why do `editVm(vm, { auto_poweron: true })` when `vm.auto_poweron` is already equal to `true`.
Isn't supposed to be `editVm(vmPool, { auto_poweron: true })` instead?

Ping @julien-f

Reply:
1. `vmPool` is not a VM
2. we could do `editPool(vmPool, { auto_poweron: true })` indeed
3. still it's working because https://github.com/vatesfr/xen-orchestra/blob/ed128e9b4bbb36943f87805a42fc352bb655d395/packages/xo-server/src/xapi/mixins/vm.mjs#L237